app/main.py

```diff
@@ -1,25 +1,85 @@
 from abc import ABC
 
 
+class SlideLimitationValidator(ABC):
+    def __init__(self,
+                 age: int,
+                 weight: (int, float),
```

Comment:
```suggestion
                 weight: int | float,
```
Or Union can be used

Reply:
Fix in all places